CosmicReco/src/PDFFit.cc

```diff
@@ -414,18 +419,17 @@ double GaussianDriftFit::TimeResidual(ComboHit const& sh, const std::vector<doub
 
   Straw const& straw = tracker->getStraw(sh.strawId());
   TwoLinePCA pca(intercept, dir, straw.getMidPoint(), straw.getDirection());
+  //int ambig = HitAmbiguity(sh, x);
 
   double traj_time = ((pca.point1() - intercept).dot(dir)) / 299.9;
-
-  double predictedTime = srep.driftDistanceToTime(sh.strawId(), pca.dca(), 0);
-
-  double hit_t0 =
-      sh.propTime() + traj_time + t0 + srep.driftTimeOffset(sh.strawId(), 0, 0, pca.dca());
-  double measuredTime = sh.time() - hit_t0;
+  double hit_t0 = t0 + traj_time + srep.driftTimeOffset(sh.strawId(), 0, 0, pca.dca());
+  
+  double predictedTime = srep.driftDistanceToTime(sh.strawId(), pca.dca(), 0) + sh.propTime() + hit_t0;
+  double measuredTime = sh.time();
 
   double resid = predictedTime - measuredTime;
 
-  return HitAmbiguity(sh, x) * resid;//(pca.s2() > 0 ? resid : -resid); // 
+  return  resid;//(pca.s2() > 0 ? resid : -resid); // 
```

Comment:
Could you add a comment to remind us why ambig isn't needed here?

Reply:
done!